android-components/components/support/license/build.gradle

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion config.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion config.minSdkVersion
+        targetSdkVersion config.targetSdkVersion
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+
+    namespace 'mozilla.components.support.license'
+}
+
+dependencies {
+    implementation ComponentsDependencies.androidx_annotation
+    implementation ComponentsDependencies.androidx_core
+    implementation ComponentsDependencies.androidx_fragment
+
+    implementation ComponentsDependencies.kotlin_coroutines
```

Comment:
Adding this line, seems to have solved the failure for `lint-lint-fenix` but should not have been needed since this component is not used in Fenix or is the dependency itself.